python/perf-kernels/06-fused-attention-fwd-transV.py

```diff
@@ -183,45 +195,49 @@ def forward(ctx, q, k, v, sm_scale):
 
 attention = _attention.apply
 
+name_to_torch_types = {
+    'fp16': torch.float16,
+    'fp8': float8
+}
 
-@pytest.mark.parametrize('Z, H, N_CTX, D_HEAD',
-                         [(4, 48, 1024, 64),
-                          (4, 48, 2048, 64),
-                          (4, 48, 4096, 64),
-                          (4, 48, 1024, 128),
-                          (4, 48, 2048, 128),
-                          (4, 48, 4096, 128),
-                          #(4, 48, 8192, 64),
-                          #(4, 48, 16384, 64)
-                          ])
-def test_op_fwd(Z, H, N_CTX, D_HEAD, dtype=torch.float16):
+@pytest.mark.parametrize('Z, H, N_CTX, D_HEAD, dtype',
+[ (*shape, dtype)
+    for shape in [(4, 48, 1024, 128),
+                  (4, 48, 2048, 128),
+                  (4, 48, 4096, 128)]
+    for dtype in ['fp16', 'fp8']])
```

Comment:
Why do we drop the 8K and 16K seq length?

Reply:
Because there is HIP memory error when using large sequence length